homeassistant/components/dsmr/sensor.py

```diff
@@ -908,7 +928,17 @@ def __init__(
         self._attr_device_class = device_class
         self._attr_native_unit_of_measurement = native_unit_of_measurement
         self._entry = entry
-        self.telegram: Telegram | None = telegram
+        self._telegram: Telegram | None = telegram  # Stores the latest telegram
+        self._obis_reference = entity_description.obis_reference
+        self._dsmr_version = entry.data[CONF_DSMR_VERSION]
+        self._value: StateType = None  # Stores the value to report
+
+        # Check if this sensor measures power (W or kW)
+        self._is_power_sensor = self.native_unit_of_measurement in {
```

Comment:
Please generalize here, indicating why the type is specialcased. Like is_averaged_sensor